pulser-core/pulser/sequence/sequence.py

```diff
@@ -405,13 +464,48 @@ def set_magnetic_field(
         # No parametrization -> Always stored as a regular call
         self._calls.append(_Call("set_magnetic_field", mag_vector, {}))
 
+    def _set_slm_mask_dmm(self, dmm_id: str, targets: set[QubitId]) -> None:
+        ntargets = len(targets)
+        detuning_map = self.register.define_detuning_map(
+            {
+                qubit: (1 / ntargets if qubit in targets else 0)
+                for qubit in self.register.qubit_ids
+            }
+        )
+        self.config_detuning_map(detuning_map, dmm_id)
```

Comment:
Beware: this call can't be stored, it will result in the same detuning map being configured twice (because there will be the `config_slm_mask` call there too). We have to separate the "storing" part from the rest, like it's done in other methods

Reply:
I am not sure I get it... I get the fact that I will end up with an additional call to config_detuning_map that will result in an error, but at the moment I am not storing the calls to config_detuning_map. Is calling the decorator `@seq_decorators.store` prior to configuring the detuning map the solution ?